piksi_tools/console/observation_view.py

```diff
@@ -36,37 +36,22 @@ class SimpleAdapter(TabularAdapter):
                ('Flags', 7)]
     font='courier'
     alignment='center'
```

Comment:
we need these format strings in order to prevent round numbers from jumping alignment